### ISSUE TITLE
[LAUNCHBLOCKER]: The confirm email address is an anti-pattern and may be linked to a critical bug

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#45020]
## Value Statement
**_As a_** user
**_I want to_** enter my email address only once
**_So that_** I can proceed with the rest of my task without being asked for redundant information. 

---
## Background Context
<details><summary>Issue Summary from Josh</summary>

The extra confirmation field for email under "additional contact information" can be deleted. @thehastingsp has some additional context here having [pushed for this in the other squad](https://dsva.slack.com/archives/C5AGLBNRK/p1656528427297259).

<img width="930" alt="Screen Shot 2022-07-28 at 11 17 25 AM" src="https://user-images.githubusercontent.com/14154792/181574720-b9e11c5f-a519-4669-b2bd-11c00cc96723.png">

On a sidenote, @scottymeyers and I caught a bug where entering into the confirm email address field post errors appearing can make the form disappear:

https://user-images.githubusercontent.com/14154792/181574823-35ae2e10-c096-4ce7-a5e5-f67623238186.mov

Caught using:
- Google Chrome
- Macbook

Curious to see if removing the field could resolve the bug?</details>

## Acceptance Criteria
- [x] _When the user is asked to enter their email address they are not required to confirm it by entering it twice in separate fields._
- [x] _When the user completes the fields they can still see the form._

## Designs and Build Notes
![image](https://user-images.githubusercontent.com/90406121/183694022-c932d80c-c510-44ef-835b-a42a38cc1807.png)

---
## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by thehastingsp** [comment]
[Here is the ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1006) intended to address this pattern that grew out of the slack discussion.

**Comment by saderagsdale** [comment]
Add AC to this story. @saderagsdale

**Comment by jerekshoe** [comment]
It's likely that you guys will end up removing the 'Confirm email' field anyways, but @scottymeyers and I fixed the issue with the form disappearing so that part should no longer be a problem

**Comment by jsokohl** [comment]
<img width="930" alt="181574720-b9e11c5f-a519-4669-b2bd-11c00cc96723" src="https://user-images.githubusercontent.com/73672031/182944047-bafad66c-dab5-4046-92dd-368957d83bbe.png">
Here's a quick sketch of what the UI should look like.

**Comment by saderagsdale** [comment]
@jsokohl @joshkimux Can you confirm if the acceptance criteria and design I added to the ticket are accurate?

**Comment by jsokohl** [comment]
Confirmed!

**Comment by saderagsdale** [comment]
@jerekshoe ok to close?

**Comment by jerekshoe** [comment]
@scottymeyers merged a PR to remove this field, so it should be good to close @saderagsdale
